packages/devtools/devtools/src/panels/client/TracingPanel/TracingPanel.tsx

```diff
@@ -112,18 +112,25 @@ export const TracingPanel = () => {
     'border-b first:border-r last:border-l',
   );
 
+  const containerRef = useRef<HTMLDivElement>(null);
+
   return (
     <PanelContainer>
-      <AnchoredOverflow.Root classNames='flex flex-col h-1/3 overflow-auto'>
-        <Table<ResourceState>
-          columns={columns}
-          data={Array.from(state.current.resources.values())}
-          currentDatum={selectedResource}
-          onDatumClick={(resourceState) => setSelectedResourceId(resourceState.resource.id)}
-          fullWidth
-        />
-        <AnchoredOverflow.Anchor />
-      </AnchoredOverflow.Root>
+      <Table.Root>
+        <Table.Viewport asChild>
+          <AnchoredOverflow.Root classNames='flex flex-col h-1/3 overflow-auto'>
```

Comment:
Not important here, but a tidier way of doing this would be to just replace all this with
`<Table.Viewport classNames='anchored-overflow flex flex-col h-1/3 overflow-auto'>`

Reply:
Correction, `overflow-anchored`.

Reply:
So we don't need the `<AnchoredOverflow.Anchor />` element if we do that?

Reply:
No you still do, that or the element that should be anchored should get the `overflow-anchor` classname.

Reply:
It’s fine to mix and match because these components do nothing more than apply these classnames, hence my wanting to remove them.